cache-filter/src/filter/http.rs

```diff
@@ -206,7 +210,11 @@ impl CacheFilter {
         let UsageReports::UsageReports(reports) = response
             .usage_reports()
             .ok_or(AuthResponseError::UsageNotFound)?;
-
+        let app_keys = response
+            .app_keys()
+            .ok_or(AuthResponseError::ListKeysMissing)?;
+        let app_id = AppIdentifier::from(AppId::from(app_keys.app_id().unwrap().as_ref()));
+        let service_id = ServiceId::from(app_keys.service_id().unwrap().as_ref());
```

Comment:
In future, those `unwraps` can become unsafe, how about an error enum for them as well?

Reply:
Please refer to the convo in the slack.

Reply:
fixed